### ISSUE TITLE
Improve the current gender field, and add fields for sex and assigned gender at birth  

### ISSUE THREAD

**Issue by cerulis64** [monicahq/monica#5802]
**Is your feature request related to a problem? Please describe.**
Gender, assigned gender at birth (agab), and sex are separate concepts, and therefore these values should be separate fields. Currently, you can create custom gender types, which is great! However, when you create a new custom gender, there is a menu selection for "sex", which is not relevant.

**Describe the solution you'd like**
Gender, assigned gender at birth, and sex should be completely separated as individual variables:
- "gender": referring to the social construct.
- "assigned gender at birth": the gender that the doctor assigned to the person when they were born.
- "sex": referring to the biological sex of the person.

It is probably a good idea to implement all the individual fields (gender, assigned gender at birth, and sex) similar to how gender is currently implemented, with options in the settings menu to add custom values. Additionally, the option for "sex" should be removed from the gender types (it is completely unrelated), and should instead be its own field.

**Comment by m-d-johnson** [comment]
What about adding a field for pronouns as part of this request?

**Comment by cerulis64** [comment]
@m-d-johnson I agree, I think a pronouns field is a very important addition, I hope it is added soon. There is already a GitHub issue regarding the addition of a pronouns field (#3513).

**Comment by puppycodes** [comment]
> **Is your feature request related to a problem? Please describe.** Gender, assigned gender at birth (agab), and sex are separate concepts, and therefore these values should be separate fields. Currently, you can create custom gender types, which is great! However, when you create a new custom gender, there is a menu selection for "sex", which is not relevant.
> 
> **Describe the solution you'd like** Gender, assigned gender at birth, and sex should be completely separated as individual variables:
> 
> * "gender": referring to the social construct.
> * "assigned gender at birth": the gender that the doctor assigned to the person when they were born.
> * "sex": referring to the biological sex of the person.
> 
> It is probably a good idea to implement all the individual fields (gender, assigned gender at birth, and sex) similar to how gender is currently implemented, with options in the settings menu to add custom values. Additionally, the option for "sex" should be removed from the gender types (it is completely unrelated), and should instead be its own field.

Gender assigned at birth is a harmful addition to this application. The Trans community is not fighting for visibility related to their gender assigned at birth, which many view as a coercive. We are interested in relaying our pronouns and gender identity. Sex at birth should not be part of the equation. The most inclusive fields are "Gender Identity" and "Pronouns". That covers every person of every identity.

**Comment by cerulis64** [comment]
@puppycodes Yes, thats a good point. Sorry, my intentions were not to be transphobic. While writing this issue that point did cross my mind, but i thought that it would be better to add detailed fields to the software to log more detailed information about people. One use case that I thought of while adding that field is that that in combination with a boolean "closeted" checkmark, and a way to track who the person is closeted/out to, the AGAB field could be useful for respecting the person's confidentiality and safety. Although if there are better alternatives for solving that issue, I am open to new ideas and suggestions.

**Comment by cerulis64** [comment]
I clicked the wrong button, closing the issue was unintentional

**Comment by cerulis64** [comment]
After thinking through the idea more clearly, I agree that the addition of AGAB fields are a harmful addition to Monica. I think that instead of splitting apart the variables, it would be better to expand and improve the current "gender types" personalization option, and remove the "sex" field from the types because it is irrelevant. A contact should be able to have the option to have multiple gender types added. I hope that the new gender types field will be flexible and accommodating to the entire gender spectrum. 
In addition to improved gender types I also think that we should add a text field for pronouns like "they/xe", "she/he/they", "fae/she", "any pronouns", etc (there is already an open issue regarding pronouns #3513).

**Comment by puppycodes** [comment]
> After thinking through the idea more clearly, I agree that the addition of AGAB fields are a harmful addition to Monica. I think that instead of splitting apart the variables, it would be better to expand and improve the current "gender types" personalization option, and remove the "sex" field from the types because it is irrelevant. A contact should be able to have the option to have multiple gender types added. I hope that the new gender types field will be flexible and accommodating to the entire gender spectrum. In addition to improved gender types I also think that we should add a text field for pronouns like "they/xe", "she/he/they", "fae/she", "any pronouns", etc (there is already an open issue regarding pronouns #3513).

Agree! Thanks so much for your response and thoughts! Love Monica and happy to see this sort of progress :)
